examples/ad9361-iiostream.c

```diff
@@ -162,7 +163,11 @@ bool cfg_ad9361_streaming_ch(struct stream_cfg *cfg, enum iodev type, int chid)
 	// Configure phy and lo channels
 	printf("* Acquiring AD9361 phy channel %d\n", chid);
 	if (!get_phy_chan(type, chid, &chn)) {	return false; }
-	wr_ch_str(chn, "rf_port_select",     cfg->rfport);
+	int ret = iio_channel_attr_write_string(chn, "rf_port_select", cfg->rfport);
+	if (ret == -EINVAL)
+		printf("Error writing %s to channel \"%s\"\nThis error can be ignored for PlutoSDR and derivatives.\n", cfg->rfport, "rf_port_select");
```

Comment:
You don't really need to use a "%s" for "rf_port_select", just write it in-line.
Also please break line before the parameters to avoid the line to be super-long.

Reply:
True. I force pushed it again.